### PR TITLE
[4.0] WebAuthn Improve new SQL, INI and XML files

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

WebAuth Improve SQL, INI and XML files:
- Reorder XML files
- Alpha order new language strings
- cleanup SQl files and rename the update files to the actual merge date

### Testing Instructions

Same as: https://github.com/joomla/joomla-cms/pull/28094

### Expected result

WebAuthn still works

### Documentation Changes Required

none

### CODE REVIEW

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 1]
```diff
@@ -4,43 +4,37 @@
...
-PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_CANCEL_LABEL="Cancel"
-
-PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR_LABEL="Authenticator added on %s"
-
-PLG_SYSTEM_WEBAUTHN_MSG_SAVED_LABEL="The label has been saved."
-PLG_SYSTEM_WEBAUTHN_MSG_DELETED="The authenticator has been removed."
-
-PLG_SYSTEM_WEBAUTHN_ERR_NO_STORED_CREDENTIAL="Cannot find the stored credentials for your login authenticator."
-PLG_SYSTEM_WEBAUTHN_ERR_CORRUPT_STORED_CREDENTIAL="The stored credentials are corrupt for your user account. Log in using another method, then remove and add again your login authenticator."
+PLG_SYSTEM_WEBAUTHN_ERR_CANNOT_FIND_USERNAME="Cannot find the username field in the login module. Sorry, Passwordless authentication will not work on this site unless you use a different login module."
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_system_webauthn.ini]:
```suggestion
PLG_SYSTEM_WEBAUTHN_ERR_CANNOT_FIND_USERNAME="Can't find the username field in the login module. Sorry, passwordless authentication will not work on this site unless you use a different login module."
```

File: plugins/system/webauthn/webauthn.xml [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-
-    <languages folder="language">
-        <language tag="en-GB">en-GB/plg_system_webauthn.ini</language>
-        <language tag="en-GB">en-GB/plg_system_webauthn.sys.ini</language>
-    </languages>
-
+<extension type="plugin" group="system" method="upgrade">
+	<name>PLG_SYSTEM_WEBAUTHN</name>
+	<author>Joomla! Project</author>
+	<creationDate>July 2019</creationDate>
```
Comment (wilsonge) [inline plugins/system/webauthn/webauthn.xml]: If you’re changing sql dates guess this should change too

**Review by wilsonge — CHANGES_REQUESTED** [review]

File: administrator/components/com_admin/sql/updates/mysql/4.0.0-2020-03-05.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS `#__webauthn_credentials`
+(
+  `id` VARCHAR(1000) NOT NULL COMMENT 'Credential ID',
+  `user_id` VARCHAR(128) NOT NULL COMMENT 'User handle',
+  `label` VARCHAR(190) NOT NULL COMMENT 'Human readable label',
+  `credential` MEDIUMTEXT NOT NULL COMMENT 'Credential source data, JSON format',
+  PRIMARY KEY (`id`,
```
Comment (alikon) [inline administrator/components/com_admin/sql/updates/mysql/4.0.0-2020-03-05.sql]:
```suggestion
   PRIMARY KEY (`id`(100)),
```

File: installation/sql/mysql/joomla.sql [hunk 4]
```diff
@@ -2283,20 +2283,17 @@ INSERT INTO `#__viewlevels` (`id`, `title`, `ordering`, `rules`) VALUES
...
-  DEFAULT COLLATE = utf8mb4_unicode_ci;
+-- Table structure for table `#__webauthn_credentials`
+--
+
+CREATE TABLE IF NOT EXISTS `#__webauthn_credentials` (
+  `id` VARCHAR(1000) NOT NULL COMMENT 'Credential ID',
+  `user_id` VARCHAR(128) NOT NULL COMMENT 'User handle',
+  `label` VARCHAR(190) NOT NULL COMMENT 'Human readable label',
+  `credential` MEDIUMTEXT NOT NULL COMMENT 'Credential source data, JSON format',
+  PRIMARY KEY (`id`),
```
Comment (alikon) [inline installation/sql/mysql/joomla.sql]:
```suggestion
   PRIMARY KEY (`id`(100)),
```

File: plugins/system/webauthn/webauthn.xml (hunk 2 above)
Comment (zero-24) [reply plugins/system/webauthn/webauthn.xml]: Done.

**Pull request closed without merging**